docs/python.rst

```diff
@@ -74,3 +74,52 @@ like this::
     - python3-requests.json
     # (other modules go here)
 
+Installing dependencies without flatpak-pip-generator using setuptools or pipenv
+--------------------------------------------------------------------------------
+
+As an alternative to flatpak-pip-generator a slightly more pythonic approach would
+be using setup.py or requirements.txt directly.
+To do this with ``setuptools`` you can do so as follows:
+
+.. code-block:: yaml
+
+  app-id: org.flatpak.PackagingDemo
+  runtime: org.freedesktop.Platform
+  runtime-version: '1.6'
+  sdk: org.freedesktop.Sdk
+  command: app.py
+  finish-args:
+  - "--share=network"
+  modules:
+  - name: cpython
```

Comment:
`python` is also in every runtime, so this seems overkill.

Reply:
So I was basing my example script from an older one. If python is shipped in the runtime by default I can remove that step.

Reply:
It has always been in every runtime AFAIK.